src/lando/tests/test_version.py

```diff
@@ -0,0 +1,17 @@
+from django.core.management import call_command
+
+
+def test_version():
+    from lando.version import version
```

Comment:
Make this a top-level import.

Reply:
My thinking was that the import could fail, so it's part of the test, but I guess it doesn't matter since it's unlikely there will ever be more than one test in this file.

Reply:
I think this is a special case, because the module that we are importing is generated by the command that is being called. I think it makes sense to try the import *after* the command is run. Unless the file is there before the command is run, in which case there's something slightly off here.

Reply:
So if this test is passing with the import at the top, it means that the command isn't being tested properly. The test should probably delete the version file, and then run the command and do the import.

Reply:
I've expanded the `version_test` to cover deleting the file and testing the import explicitly.